channels/urbdrc/client/libusb/libusb_udevman.c

```diff
@@ -347,73 +347,30 @@ static BOOL udevman_unregister_all_udevices(IUDEVMAN* idevman)
 	return TRUE;
 }
 
-static BOOL udevman_parse_device_addr(const char* str, size_t maxLen, UINT8* id1, UINT8* id2,
-                                      char sign)
+static BOOL udevman_parse_device_id_addr(char** str, unsigned long* id1, unsigned long* id2,
```

Comment:
Would suggest making the `id1` and `id2` of type `UINT16` and use an internal variable for `strtoul` (add a range arguement too, so too large values fail the parsing)